apps/examples/smartfs/fs_performance/fs_performance_main.c

```diff
@@ -0,0 +1,461 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <tinyara/timer.h>
+
+/****************************************************************************
+ * Pre Processor Definitions
+ ****************************************************************************/
+
+#define ROOT				"/mnt/"
```

Comment:
How about change name root to MNT. I think people can be confused to name.

Reply:
How about using "SMARTFS_ROOT" since the example is only for SmartFs FileSystem? @pillip8282 @dineshdas-samsung

Reply:
CONFIG_MOUNT_POINT is already defined in fs.h

Reply:
@Taejun-Kwon I think using CONFIG_MOUNT_POINT is good.

Reply:
Thankyou All. Updated